RadixWallet/Features/SettingsFeature/ManualAccountRecoveryScan/ManualAccountRecoveryLedgerCoordinator+Reducer.swift

```diff
@@ -0,0 +1,98 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - ManualAccountRecoveryLedgerCoordinator
+public struct ManualAccountRecoveryLedgerCoordinator: Sendable, FeatureReducer {
+	public typealias Store = StoreOf<Self>
+
+	// MARK: - State
+
+	public struct State: Sendable, Hashable {
+		public var root: LedgerHardwareDevices.State = .init(context: .accountRecovery)
```

Comment:
Need value indicating if it is an Olympia or Babylon scan - which will AccountRecoveryScanCoordinator requires - which affects if a CAP26 or BIP44Olymlia derivation path is used.